bpf/include/vmlinux.h

```diff
@@ -1435,6 +1437,37 @@ struct pt_regs {
 	long unsigned int ss;
 };
 
+#elif defined(__TARGET_ARCH_arm64)
```

Comment:
```suggestion
#elif defined(__TARGET_ARCH_arm64)
// see: https://github.com/cilium/tetragon/issues/786
```

I've tried to summarize the discussion (both here and offline in a comment). Let's use this approach for now, but add a comment that points to the issue so that we can fix it later.

Reply:
See this: https://github.com/cilium/tetragon/pull/734/commits/c996118abb3d40a73c827705e87bcf448897b973#diff-e7b63fd817e31c67015e2c425624987ad9ef27a734909c92d9cb218b23fc8904R1441-R1443

Thanks!! :)